README.md

```diff
@@ -6,53 +6,54 @@
-		- [Configuring SSE node connections](#configuring-sse-node-connections)
-		- [Configuring SSE legacy emulations](#configuring-sse-legacy-emulations)
-		- [Configuring the event stream](#configuring-the-event-stream)
-	- [REST server setup](#rest-server-setup)
-	- [Storage setup](#setup-storage)
-	- [Database connectivity setup](#database-connectivity-setup)
-		- [SQLite database](#sqlite-database)
-		- [PostgreSQL database](#postgresql-database)
-	- [Admin server setup](#admin-server-setup)
+  - [RPC server setup](#rpc-server-setup)
+  - [SSE server setup](#sse-server-setup)
+    - [Configuring SSE node connections](#configuring-sse-node-connections)
+    - [Configuring SSE legacy emulations](#configuring-sse-legacy-emulations)
+    - [Configuring the event stream](#configuring-the-event-stream)
+  - [REST server setup](#rest-server-setup)
+  - [Storage setup](#setup-storage)
+  - [Database connectivity setup](#database-connectivity-setup)
+    - [SQLite database](#sqlite-database)
+    - [PostgreSQL database](#postgresql-database)
+  - [Admin server setup](#admin-server-setup)
 - [Running and Testing the Sidecar](#running-and-testing-the-sidecar)
-	- [Prerequisites](#prerequisites)
-   - [Running the Sidecar](#running-the-sidecar)
-   - [Testing the Sidecar](#testing-the-sidecar)
+  - [Prerequisites](#prerequisites)
+  - [Running the Sidecar](#running-the-sidecar)
+  - [Testing the Sidecar](#testing-the-sidecar)
 - [Swagger Documentation](#swagger-documentation)
 - [OpenAPI Specification](#openapi-specification)
 - [Troubleshooting Tips](#troubleshooting-tips)
-	- [Checking liveness](#checking-liveness)
-	- [Checking the node connection](#checking-the-node-connection)
-	- [Diagnosing errors](#diagnosing-errors)
-	- [Monitoring memory consumption](#monitoring-memory-consumption)
-	- [Ensuring sufficient storage](#ensuring-sufficient-storage)
-	- [Inspecting the REST API](#inspecting-the-rest-api)
-	- [Limiting concurrent requests](#limiting-concurrent-requests)
+  - [Checking liveness](#checking-liveness)
+  - [Checking the node connection](#checking-the-node-connection)
+  - [Diagnosing errors](#diagnosing-errors)
+  - [Monitoring memory consumption](#monitoring-memory-consumption)
+  - [Ensuring sufficient storage](#ensuring-sufficient-storage)
+  - [Inspecting the REST API](#inspecting-the-rest-api)
+  - [Limiting concurrent requests](#limiting-concurrent-requests)
 
 ## Summary of Purpose
 
 The Casper Sidecar is an application running in tandem with the node process. It allows subscribers to monitor a node's event stream, query stored events, and query the node's JSON RPC API, thus receiving faster responses and reducing the load placed on the node. Its primary purpose is to:
 
-* Offload the node from broadcasting SSE events to multiple clients.
-* Provide client features that aren't part of the nodes' functionality, nor should they be.
+- Offload the node from broadcasting SSE events to multiple clients.
```

Comment:
Automatic lint rules in VS Code change asterix bullet points to dash ones. Since we had no unification in this regard, I'll just leave the unified dash approach.